platform-common-core/src/main/java/com/softicar/platform/common/core/number/formatter/BigDecimalScaleApplier.java

```diff
@@ -0,0 +1,92 @@
+package com.softicar.platform.common.core.number.formatter;
+
+import java.math.BigDecimal;
+import java.math.RoundingMode;
+
+/**
+ * Applies a given scale to {@link BigDecimal} values.
+ *
+ * @author Oliver Richers
+ */
+public class BigDecimalScaleApplier {
+
+	private Integer scale;
+	private RoundingMode roundingMode;
+
+	public BigDecimalScaleApplier() {
+
+		this.scale = null;
+		this.roundingMode = null;
+	}
+
+	/**
+	 * Defines the desired scale of the values, i.e. the number of decimal
+	 * places.
+	 * <p>
+	 * If <i>null</i> is provided as scale (which is the default), no changes
+	 * will be applied to the value.
+	 *
+	 * @param scale
+	 *            the scale; or <i>null</i> to disable scaling
```

Comment:
"disable scaling" - what does that mean? Seems to imply that a previously-scaled value could become unscaled if `null` is given here.
suggestions:
- `to perform no scaling`
- `to omit scaling`
- `to skip scaling`
- `to not apply scaling`
- `to not change scaling`
- ...

Reply:
ok